Automattic/wp-calypso
# Migrate post schedule styles

#### Changes proposed in this Pull Request

Migrate styles for post schedule block.

#### Testing instructions

Go to http://calypso.localhost:3000/devdocs/blocks/post-schedule and confirm that the PostSchedule block Is styled correctly.

##### Unstyled
<img width="329" alt="screen shot 2018-10-03 at 2 45 24 am" src="https://user-images.githubusercontent.com/10561050/46371293-8c9b3a00-c6ba-11e8-9aff-241bf0ec6b15.png">

##### Styled
<img width="323" alt="screen shot 2018-10-03 at 2 46 18 am" src="https://user-images.githubusercontent.com/10561050/46371303-91f88480-c6ba-11e8-9cc3-e3e0b423db0a.png">

#27515
